core/src/main/resources/db/cgds.sql

```diff
@@ -727,3 +729,10 @@ CREATE TABLE `info` (
 );
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
 INSERT INTO info VALUES ('1.1.0');
+
+--1.2.0 specific changes:
```

Comment:
I would put all the db changes into the corresponding tables directly. We don't have to separate the 1.2.0 specific changes in one block (they are already in migration.sql).